src/TLuaInterpreter.cpp

```diff
@@ -13493,98 +13493,98 @@ void TLuaInterpreter::initLuaGlobals()
 
     error = luaL_dostring(pGlobalLua, "require \"rex_pcre\"");
     if (error != 0) {
-        string e = "no error message available from Lua";
+        QString e = tr("no error message available from Lua");
         if (lua_isstring(pGlobalLua, -1)) {
-            e = "Lua error:";
+            e = tr("Lua error:");
             e += lua_tostring(pGlobalLua, -1);
         }
-        QString msg = "[ ERROR ] - Cannot find Lua module rex_pcre.\n"
-                      "Some functions may not be available.\n";
-        msg.append(e.c_str());
+        QString msg = tr("[ ERROR ] - Cannot find Lua module rex_pcre.\n"
```

Comment:
FTR appending `QString`s is less efficient than creating them with all the elements at once, it would have been better here to have changed lines 13501 to 13503 to:
```
         QString msg = tr("[ ERROR ] - Cannot find Lua module rex_pcre.\n" 
                          "Some functions may not be available.\n
                          "%1").arg(e); 
```

Reply:
Noted as a general issue as: https://github.com/Mudlet/Mudlet/issues/2123